typescript-test-samples/launchdarkly-lambda-dynamodb/metadata.json

```diff
@@ -0,0 +1,34 @@
+{
+  "title": "Typescript: LaunchDarkly, AWS Lambda and DynamoDB Example",
+  "description": "This project contains examples of integrating LaunchDarkly feature flags into Lambda.",
+  "content_language": "English",
+  "language": "TypeScript",
+  "type": ["Integration"],
+  "diagram": "/img/LaunchDarkly-AWS-diagram.png",
+  "framework": "CDK",
+  "services": ["lambda", "dynamodb"],
+  "git_repo_url": "https://github.com/aws-samples/serverless-test-samples",
+  "pattern_source": "AWS",
+  "pattern_detail_tabs": [
+    {
+      "title": "Lambda using LaunchDarkly",
```

Comment:
Can we call these Lambda _functions_?  I.E. "Lambda function using LaunchDarkly"

Reply:
Pushed a fix